app/src/main/java/com/secuso/privacyFriendlyCodeScanner/qrscanner/generator/QRGeneratorUtils.java

```diff
@@ -128,7 +148,7 @@ private static File writeToFile(File path, Bitmap image) {
         // if multiple codes are generated on the same day.. name them with numbers
         for(int i = 2; result.exists(); i++) {
             sb.delete(17, sb.length());
-            sb.append(" (").append(i).append(").png");
+            sb.append("_s(").append(i).append(").png");
```

Comment:
what is the s doing here?

Reply:
"s" for "scan" ;)
I removed the "s" in my last commit.